dsl_parser/rel_graph.py

```diff
@@ -904,7 +904,7 @@ def _containing_groups(self, node_id):
             succ = graph.succ[node_id]
             if succ:
                 assert len(succ) == 1
-                node_id = succ.keys()[0]
+                node_id = list(succ.keys())[0]
```

Comment:
didnt really want to do a `.pop` for just the key. Meh, this might as well just be this